hibernate-reactive-core/src/main/java/org/hibernate/reactive/vertx/VertxInstance.java

```diff
@@ -7,10 +7,11 @@
 
 import io.vertx.core.Vertx;
 
+import org.hibernate.reactive.pool.impl.ManagedSqlClientPool;
 import org.hibernate.service.Service;
 
 /**
- * Used by {@link org.hibernate.reactive.pool.impl.SqlClientPool}
+ * Used by {@link ManagedSqlClientPool}
```

Comment:
Probably better to keep the FQN here instead of using `import`, since you'll sometimes get warnings about an unused import (depending on the tool).

Reply:
+1 that wasn't intentional. IDE magic..